.travis.yml

```diff
@@ -20,9 +20,7 @@ install:
   - gem install jekyll -v 3.2.1
 
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION test
-  - sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues
-  - sbt ++$TRAVIS_SCALA_VERSION docs/makeSite
+  - sbt ++$TRAVIS_SCALA_VERSION validate
```

Comment:
This use of `validate` neglects that we only build the docs on one version.  We'll need to rethink this when the merge conflict from #4 hits us.